internal/cmd/operator-sdk/generate/bundle/cmd.go

```diff
@@ -138,6 +139,7 @@ func (c *bundleCmd) addFlagsTo(fs *pflag.FlagSet) {
 		"Names of service accounts, outside of the operator's Deployment account, "+
 			"that have bindings to {Cluster}Roles that should be added to the CSV")
 	fs.BoolVar(&c.overwrite, "overwrite", true, "Overwrite the bundle's metadata and Dockerfile if they exist")
+	fs.BoolVar(&c.ignoreIfOnlyCreatedAt, "ignore-if-only-createdAt", false, "Ignore if only createdAt is changed")
```

Comment:
Command line flags typically don't use camelCase. I would probably call this something like `ignore-if-only-created-at-changed`.

Reply:
Resolved in https://github.com/operator-framework/operator-sdk/pull/6419/commits/9a2f231bbab4a903dbdd6b48f0d6e78d76309232